src/HyperFormula.ts

```diff
@@ -1093,29 +1091,29 @@ export class HyperFormula implements TypedEmitter {
    *  ['1', '2'],
    * ]);
    *
-   * // choose the address and assign it to a variable
-   * const address = { col: 0, row: 0, sheet: 0 };
+   * // top left corner
+   * const address1 = { col: 0, row: 0, sheet: 0 };
+   * // bottom right corner
+   * const address2 = { col: 1, row: 0, sheet: 0 };
    *
    * // should return 'true' for this example, it is possible to set content of
    * // width 2, height 1 in the first row and column of sheet 0
-   * const isSettable = hfInstance.isItPossibleToSetCellContents(address, 2, 1);
+   * const isSettable = hfInstance.isItPossibleToSetCellContents({start: address1, end: address2});
    * ```
    *
    * @category Cells
    */
-  public isItPossibleToSetCellContents(topLeftCornerAddress: SimpleCellAddress, width: number = 1, height: number = 1): boolean {
-    validateArgToType(width, 'number', 'width')
-    validateArgToType(height, 'number', 'height')
+  public isItPossibleToSetCellContents(address: SimpleCellAddress | SimpleCellRange): boolean {
+    let range
+    if(isSimpleCellAddress(address)) {
+      range = new AbsoluteCellRange(address, address)
+    } else {
+      range = new AbsoluteCellRange(address.start, address.end)
+    }
```

Comment:
Are we sure if it's not Simple Cell Address it will always be SimpleCellRange then? In JS it may be any object. Maybe we should keep the `@throws` and validate the input?

```suggestion
    } else if (isSimpleCellRange(address) {
      range = new AbsoluteCellRange(address.start, address.end)
    } else {
      throw new ExpectedValueOfTypeError()
    }
```

May apply to all other methods

Reply:
We are not checking the types of any other composite types, though.